src/lib/genesis_ledger_helper/genesis_ledger_helper.ml

```diff
@@ -75,8 +75,8 @@ let assert_filehash_equal ~file ~hash ~logger =
   else
     let%map () = Unix.rename ~src:file ~dst:(file ^ ".incorrect-hash") in
     [%log error]
-      "Verification failure: downloaded $file and expected SHA3-256 = $hash \
-       but it had $computed_hash"
+      "Verification failure: downloaded $path and expected SHA3-256 = \
```

Comment:
Technically it should go to a separate PR, but I'm happy to keep it as is

Reply:
100% agree, i kept it here because I needed to look at the hash while verifying